cfgov/housing_counselor/results_archiver.py

```diff
@@ -0,0 +1,21 @@
+import json
+from zipfile import ZIP_DEFLATED, ZipFile
+
+
+json_file_path = './HUD_approved_housing_counselors.json'
+
+
+# By request from stakeholders in the Office of Innovation, save a
+# list of the full results we receive from the HUD API, in case of
+# future enforcement actions.
+
+# As an MVP, we're saving the complete JSON response, zipped. As a future
+# enhancement, we could consider improving the format of the data,
+# e.g. as CSV.
+def save_list(counselors, destination_path):
+    with open(json_file_path, 'w') as f:
+        f.write(json.dumps(counselors))
```

Comment:
One more minor suggestion: won't this code unnecessarily leave behind the HUD_approved_housing_counselors.json file on disk? You could use [`ZipFile.writestr`](https://docs.python.org/2/library/zipfile.html#zipfile.ZipFile.writestr) to directly write the JSON with something like this, avoiding the need to generate that temporary uncompressed file:

```py
zip.writestr(json_file_path, json.dumps(counselors))
```

Reply:
Nice. Added.